subgraphs/abyssmal/schema.graphql

```diff
@@ -9,5 +9,69 @@ type Approvals @entity {
   approved: String!
 
   " NFT tokenId "
-  tokenId: Int!
+  tokenId: BigInt!
+}
+
+type ApprovalForAllTx @entity {
+  " { transaction hash - logIndex } "
+  id: ID!
+
+  " Address of Owner "
+  owner: String!
```

Comment:
What is your opinion on difference between `String` and `Bytes` in these schema's to represent the primitive type `address` in Solidity?

Reply:
That's a very good question and I don't have an answer now. I'll look into it

Reply:
Thank you, changed to bytes for addresses.